src/main/resources/application.properties

```diff
@@ -25,8 +25,14 @@ logging.file=${logging.path}/app.log
 management.endpoints.web.base-path=/admin
 management.endpoints.jmx.exposure.exclude=*
 management.endpoints.web.exposure.include=*
+management.endpoint.health.show-details=always
 
 javamelody.management-endpoint-monitoring-enabled=true
+
+# Health check settings
+health.databaseConnection.timeoutMillis=120000
```

Comment:
To long because postgresql will throw exception after 30 seconds (default setting for request timeout)
The same in src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

Reply:
Please ignore. This is true only in case of refused connection